extensions/sat/json-schema/schema.json

```diff
@@ -50,10 +46,22 @@
                 "descending",
                 "geostationary"
               ]
+            },
+            "sat:absolute_orbit": {
+              "type": "integer",
+              "minimum": 1
+            },
+            "sat:relative_orbit": {
+              "type": "integer",
+              "minimum": 1
```

Comment:
Just wondering: Can relative orbit be 0 (or even negative)? Can absolute orbit be 0?

Reply:
not in my knowledge, it is always starting from 1 in the beginning of the repeat cycle.